### PR TITLE
Add a hook for weasyprint

### PR DESCRIPTION
**Pull request by Zitrax**
Add the needed html5_ua.css datafile.

Note I don't know the weasyprint package very well, but at least this file is required for generating a pdf in the simple use case.

### CODE REVIEW

**Comment by htgoebel** [comment]
Thanks for this hook. We'd like to integrate it, but it needs some work (see line comments).

File: PyInstaller/hooks/hook-weasyprint.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import os
```
Comment (htgoebel) [inline PyInstaller/hooks/hook-weasyprint.py]: Please add the copyright header like in the other hook files.
Please add comments containing the URL of the package and about what version and what OS this hook is tested with

File: PyInstaller/hooks/hook-weasyprint.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+from distutils.sysconfig import get_python_lib
+
+datas = [
```
Comment (htgoebel) [inline PyInstaller/hooks/hook-weasyprint.py]: Please use PyInstaller.utils.hooks.hookutils.collect_data_files

**Comment by Zitrax** [comment]
Thanks for the notes, I'll fix it up tomorrow.

**Comment by matysek** [comment]
Thanks.

**Pull request merged**